integration-tests/hybrid-smoke-tests/pom.xml

```diff
@@ -13,7 +13,7 @@
     <artifactId>quarkus-hilla-hybrid-smoke-tests</artifactId>
     <name>Quarkus - Hilla - Hybrid Smoke Tests</name>
     <properties>
-        <vaadin.version>24.2.4</vaadin.version>
+        <vaadin.version>24.3.0.alpha6</vaadin.version>
```

Comment:
Is here always a specific version pinned instead of the latest snapshot?

Reply:
Good point. We can pin a snapshot version to eagerly find issues in hybrid module

Reply:
done